src/core/agent/services/ipexCommunicationService.test.ts

```diff
@@ -202,6 +207,19 @@ jest.mock("signify-ts", () => ({
   Serder: jest.fn().mockImplementation(() => {
     return {};
   }),
+  Saider: {
+    saidify: jest.fn().mockImplementation(() => {
+      return ["mockSaid", { d: "mockKed" }];
+    }),
+  },
+  Siger: jest.fn().mockImplementation(() => {
+    return {};
+  }),
+  messagize: jest.fn().mockImplementation(() => {
+    return {};
+  }),
+  d: jest.fn().mockImplementation(() => "d"),
+  b: jest.fn().mockImplementation(() => "b"),
```

Comment:
I'm going to ask this again. If we are using proper values in our fixtures/mocks, why do we need to mock all of these things? Can we not just use the real values?

Reply:
so you want mock funct d,b with real value return right?

Reply:
so you want mock return of these functions with real values?

Reply:
No, I am asking why we need to mock these functions at all. If our fixtures are using real data, then Signify primitive functions like this do not need to be mocked.

`b` for example just converts from string to bytes, and `d` the opposite. Can't we just leave Signify do this without mocking it? We should only mock what's necessary, such as things returned from KERIA

Reply:
If i dont mock these functions, my test cases will stuck when the logic run to lines using them

Reply:
Lets discuss on slack

Reply:
(of course only possible if spying and not mocking, complete oversight on my part. I still think there's potential value in spying there, but in the future)